met-web/src/styles/Theme.ts

```diff
@@ -62,6 +62,7 @@ export const BaseTheme = createTheme({
         MuiButton: {
             styleOverrides: {
                 root: {
+                    lineHeight: '1.1rem',
```

Comment:
can we remove the style overrides here and just add them to PrimaryButton and SecondaryButton

Reply:
sure thing

Reply:
have we also removed any other style overrides here and moved them to the atomic components?